test/membrane_rtmp_plugin/rtmp_source_bin_test.exs

```diff
@@ -0,0 +1,119 @@
+defmodule Membrane.RTMP.SourceBin.IntegrationTest do
+  use ExUnit.Case
+
+  import Membrane.Testing.Assertions
+
+  require Logger
+
+  alias Membrane.RTMP.Source.TcpServer
+  alias Membrane.Testing
+
+  @input_file "test/fixtures/testsrc.flv"
+  @port 1935
+  @local_ip "127.0.0.1"
+  @stream_key "ala2137"
+  @rtmp_stream_url "rtmp://#{@local_ip}:#{@port}/app/#{@stream_key}"
+
+  test "SourceBin outputs the correct number of audio and video buffers" do
+    test_process = self()
+
+    options = %TcpServer{
+      port: @port,
+      listen_options: [
+        :binary,
+        packet: :raw,
+        active: false,
+        ip: @local_ip |> String.to_charlist() |> :inet.parse_address() |> elem(1)
+      ],
+      socket_handler: fn socket ->
+        options = [
+          module: Membrane.RTMP.Source.TestPipeline,
+          custom_args: [socket: socket, test_process: test_process],
+          test_process: test_process
+        ]
+
+        Testing.Pipeline.start_link(options)
+      end
+    }
+
+    {:ok, _tcp_server} = TcpServer.start_link(options)
+    ffmpeg_task = Task.async(&start_ffmpeg/0)
+
+    pipeline = await_pipeline_started()
+
+    assert_pipeline_playback_changed(pipeline, :prepared, :playing)
+
+    assert_buffers(%{
+      pipeline: pipeline,
+      sink: :video_sink,
+      stream_length: 3000,
+      buffers_expected: div(3000, 42)
```

Comment:
Can you make that `42` and `23` below module attributes as something like this `@video_frame_duration_ms` and `audio_frame_duration_ms`?